realm/services/google_oauth/incoming_webhooks/oauth/source.js

```diff
@@ -0,0 +1,98 @@
+
```

Comment:
Looks like this is specific to YouTube. Can you add a comment to the top indicating what this function is doing?

Reply:
It's currently specific to YouTube, but it will be generic. (There will be a configuration object in the database for each oauth provider, I think)